easybuild/easyconfigs/d/DSRC/DSRC-2.0rc-linux-64-bit.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'Binary'
+
+name = 'DSRC'
+version = '2.0rc'
+versionsuffix = '-linux-64-bit'
+
+homepage = 'http://sun.aei.polsl.pl/dsrc'
+description = """DNA Sequence Reads Compression is an application designed for compression of data files
+ containing reads from DNA sequencing in FASTQ format. The amount of such files can be huge, e.g., a few
+ (or tens) of gigabytes, so a need for a robust data compression tool is clear. Usually universal compression
+ programs like gzip or bzip2 are used for this purpose, but it is obvious that a specialized tool can work better."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# fi. http://sun.aei.polsl.pl/dsrc/download/2.0rc/dsrc
+sources = [name.lower()]
+source_urls = [homepage + '/download/%(version)s/']
```

Comment:
I'd rather have the `homepage` bit copy-pasted than this string concatenation here...